service/participants.js

```diff
@@ -17,11 +17,24 @@ let service = {};
 
 service._nodemailer = nodemailer;
 
+function participantFromRow(row) {
```

Comment:
that is also possible but then the model will have to know the schema of the table. I put it in that file to avoid having table knowledge in the model